controllers/configmap_controller.go

```diff
@@ -218,10 +219,6 @@ func (r *ConfigMapReconciler) ensureInstancesAreUpToDate(instances []*instances.
 // deconfigures the instances associated with them.
 func (r *ConfigMapReconciler) deconfigureInstances(instances []*instances.InstanceInfo, nodes *core.NodeList) error {
 	for _, node := range nodes.Items {
-		// Only looking at BYOH nodes
-		if _, present := node.Annotations[BYOHAnnotation]; !present {
-			continue
-		}
```

Comment:
This implies the method doc needs to be updated to state that that input nodes are assumed to be Windows BYOH